### ISSUE TITLE
Results are not getting reproduced of Office31

### ISSUE THREAD

**Issue by shreejalt** [NaJaeMin92/FixBi#12]
Hi @NaJaeMin92,
I tried using the repo but I was unable to reproduce the results office dataset. My SDM accuracy got decreased to 51% and TDM is stagnant at  6%.

**Comment by NaJaeMin92** [comment]
Hi @shreejalt 

As I mentioned in https://github.com/NaJaeMin92/FixBi/issues/9#issue, please use baseline weights such as DANN, MSTN and check your hyper-parameters according to the performance of the baseline models.

**Comment by Bang2hen1iu** [comment]
> 

question. I follow #9, but neither source-only pretrained weight nor DANN pretrained weight gets good result. Is pretrained weight means the weight of encoder? Or perhaps the whole baseline architecture should be trained together with fixbi?

**Comment by codezjt** [comment]
> Hi @NaJaeMin92, I tried using the repo but I was unable to reproduce the results office dataset. My SDM accuracy got decreased to 51% and TDM is stagnant at 6%.

Have you resolve the question? I got the same result.

**Comment by WANGBohaO-jpg** [comment]
I got the same result.
